### PR TITLE
White 'n clean design (based on new logo)

### PR DESCRIPTION
**Pull request by andrenarchy**
* new logo
* white navbar
* light gray background
* full-width
* new `element-position` directive determines position of element in viewport
* sticky subnav bar in article views
* new discussion style

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -57,8 +57,8 @@ function js (watch) {
   var shim = require('browserify-shim');
   var watchify = require('watchify');
 
-  var browserify_args = _.extend(watchify.args, {debug: true});
-  var bundler = browserify('./src/js/index.js', browserify_args);
+  var browserifyArgs = _.extend(watchify.args, {debug: true});
```
Comment (nschloe) [inline gulpfile.js]: Unrelated to this PR, but: Do we really need `debug: true` unconditionally?

File: src/js/directives/elementPosition.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var angular = require('angular');
+
+module.exports = function(app) {
+
+  app.directive('elementPosition', [
```
Comment (nschloe) [inline src/js/directives/elementPosition.js]: A little documentation would be nice. What is the directive, what does it do, how do I use it?

File: src/js/directives/elementSize.js [hunk 3]
```diff
@@ -1,45 +1,61 @@
...
 
-          // return if unchanged
-          if (size.height === height && size.width === width) {return;}
+var angular = require('angular');
 
-          size.height = height;
-          size.width = width;
+module.exports = function(app) {
 
+  app.directive('elementSize', [
```
Comment (nschloe) [inline src/js/directives/elementSize.js]: Likewise.

**Comment by nschloe** [comment]
Shouldn't "1 answer" and "Options" be next to each other?
lulz

**Comment by nschloe** [comment]
Is the bootstrap "danger red" adapted to our color scheme yet, i.e., #ff0000/#ff4000?
red

File: gulpfile.js (hunk 1 above)
Comment (andrenarchy) [inline gulpfile.js]: `debug` generates source maps. Perhaps we can disable it in the production builds?

File: src/js/directives/elementPosition.js (hunk 2 above)
Comment (andrenarchy) [inline src/js/directives/elementPosition.js]: Documentation added.

File: src/js/directives/elementSize.js (hunk 3 above)
Comment (andrenarchy) [inline src/js/directives/elementSize.js]: Documentation added.

**Comment by andrenarchy** [comment]
I can't reproduce the "1 answer" and "options" bug. It looks fine with Chrome here:
screenshot from 2015-10-03 14 57 15
Did you do any window resizing? Perhaps it's a float issue (missing `clearfix`)?

**Comment by nschloe** [comment]
@andrenarchy The answer/options bug occurs only when a reply is inserted, before you reload the page. To reproduce, just add a comment anywhere, plus a reply. Observe the quirked output. Reload the page and observe how everything looks normal.

**Comment by nschloe** [comment]
The "x replies/options" bug can be reproduced on Alex's computer (chrome).

**Pull request merged**